### ISSUE TITLE
gazebo_version_helpers failed to build

### ISSUE THREAD

**Issue by cmakelabs** [JenniferBuehler/gazebo-pkgs#65]
Hi
I am trying to build the packages using `catkin build`, I am using `ros-noetic` on ubuntu 22.04, with `gazebo-classic`, however the build fails due to issue in `gazebo_version_helpers`. Could you please help?
![Screenshot from 2024-01-07 19-21-55](https://github.com/JenniferBuehler/gazebo-pkgs/assets/153651682/86d768f8-1827-47bd-8737-a37ce8e37786)

**Comment by JenniferBuehler** [comment]
Hi,

The last version tested was noetic on Ubuntu 20.04. I have it on my schedule to remove gazebo_version_helpers some time in the next couple of months. In case you end up doing it before, it would be nice if you could post a PR. It's basically just removing the #ifdef statements, replace the function calls by the gazebo ones, and drop support for old gazebo versions.
Or just fixing the compiler issue at hand.

Sorry I can't be of more help right now, I currently lack the time to update these packages. But it's on my horizon.

**Comment by cmakelabs** [comment]
It seems that we both lack the time for this simple task :)

**Comment by JenniferBuehler** [comment]
Yes I'm sorry, it's been so busy here. It is moving up my to-do list though, I'm working my way through to get around to looking after my packages (long overdue). I'm planning to migrate all ROS packages to ROS2 within the next 1-2 months. Fingers crossed this timeline works out ;)

**Comment by linhezi** [comment]
> Yes I'm sorry, it's been so busy here. It is moving up my to-do list though, I'm working my way through to get around to looking after my packages (long overdue). I'm planning to migrate all ROS packages to ROS2 within the next 1-2 months. Fingers crossed this timeline works out ;)

Hi,the same qusetion happend when I use this plugin on Melodic . I’m wondering if there’s still a chance to crack this little riddle we’re facing here in 2025?
